packages/calcite-components/src/components/split-button/split-button.scss

```diff
@@ -126,13 +128,15 @@
 }
 
 @include disabled() {
-  .divider-container {
-    @apply opacity-disabled;
-  }
-
   calcite-dropdown > calcite-button {
     @apply pointer-events-none;
   }
+  &:host([appearance="outline-fill"]),
```

Comment:
I think this selector is not needed anymore since your changes above already set the background color to `transparent`.

Reply:
This is needed because specificity for disabled buttons or else a the divider appears like a light line.